### PR TITLE
added romeve Journal entry and Note Management skeleton

### LINKED ISSUE #4: Note Management
As a blog connoisseur I would like to be able to manage my personal notes about a particular blog post so that I can keep track of my thoughts about the post.

**Given** the user is viewing the Post Detail menu  
**When** they select the option to manage notes
**Then** they should be presented with the Note Management menu

The menu should contain the following options:

- List Notes
- Add Note
- Remove Note
- Return

### LINKED ISSUE #42: Remove Journal Entry
As an introspective person, I would like the ability to remove a journal entry so that I can get something I regret writing out of my life.

**Given** the user is viewing the Journal Management menu  
**When** they select the option to remove a journal entry  
**Then** they should be presented with a list of entries to choose from  

**Given** the user chooses an entry  
**When** they enter the selection and hit enter  
**Then** the journal entry should be removed from the database

### PR DESCRIPTION
**Pull request by wizldrv**
# Description
added remove and delete Journal entry
added note management skeleton
Fixes #42 #4 
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by k-sarito — DISMISSED** [review]

**Comment by k-sarito** [comment]
holy guacamole

File: TabloidCLI/Repositories/NoteRepository.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TabloidCLI.Repositories
+{
+    internal class Class1
```
Comment (joshdbarton) [inline TabloidCLI/Repositories/NoteRepository.cs]: guess this is unnecessary?

**Review by joshdbarton — CHANGES_REQUESTED** [review]

File: TabloidCLI/Repositories/NoteRepository.cs (hunk 1 above)
Comment (wizldrv) [reply TabloidCLI/Repositories/NoteRepository.cs]: oops. I went too fast

Comment (wizldrv) [reply TabloidCLI/Repositories/NoteRepository.cs]: didn't save. before i pushed.... 😵‍💫

Comment (wizldrv) [reply TabloidCLI/Repositories/NoteRepository.cs]: Should be fixed now

**Review by joshdbarton — APPROVED** [review]

**Review by k-sarito — APPROVED** [review]

**Pull request merged**